content/en/opentelemetry/troubleshooting.md

```diff
@@ -84,17 +101,135 @@ To troubleshoot:
+
+#### For OpenTelemetry Collector
+
+Use the `transform` processor to set your host tags as OTLP attributes. For example, to add environment and team tags:
+
+```yaml
+processors:
+  transform:
+    trace_statements:
+      - context: resource
+        statements:
+          # OpenTelemetry semantic conventions
+          - set(attributes["deployment.environment.name"], "prod")
+          # Datadog-specific host tags
+          - set(attributes["ddtags"], "env:prod,team:backend")
+...
+```
+
+This approach combines OpenTelemetry semantic conventions with Datadog-specific host tags to ensure proper functionality in both OpenTelemetry and Datadog environments.
+
+## Unable to map 'team' attribute to Datadog team tag
+
+**Symptom**: The team tag is not appearing in Datadog for logs and traces, despite being set as a resource attribute in OpenTelemetry configurations.
+
+**Resolution**:
+
+Use the OpenTelemetry Collector's transform processor to map the team resource attribute to the ddtags attribute:
+
+```yaml
+processors:
+  transform/datadog_team_tag:
+    metric_statements:
+      - context: datapoint
+        statements:
+          - set(attributes["ddtags"], Concat(["team:", resource.attributes["team"]],""))
+    log_statements:
+      - context: log
+        statements:
+          - set(attributes["ddtags"], Concat(["team:", resource.attributes["team"]],""))
+    trace_statements:
+      - context: span
+        statements:
+          - set(attributes["ddtags"], Concat(["team:", resource.attributes["team"]],""))
+```
+
+<div class="alert alert-info">Replace <code>resource.attributes["team"]</code> with the actual attribute name if different in your setup (for example, <code>resource.attributes["arm.team.name"]</code>).</div>
+
+To verify the configuration:
+
+1. Restart the OpenTelemetry Collector to apply the changes.
+2. Generate test logs and traces.
+3. Check if the team tag appears in your Datadog logs and traces.
+4. Verify that the team tag functions as expected in filtering and dashboards.
+
+### Container tags not appearing in Containers page
+
+**Symptom**: Container tags are not appearing on the Containers page in Datadog, which affects container monitoring and management capabilities.
+
+**Resolution**:
+
+When using OTLP ingestion in the Datadog Agent, you need to set specific resource attributes to ensure proper container metadata association. For more information, see [Resource Attribute Mapping][4].
+
+To verify the configuration:
+
+1. Check the raw trace data to confirm that container IDs and tags are properly translated into Datadog format (for example, `container.id` should become `container_id`).
+2. Verify that container metadata appears on the Containers page.
+
+## Missing metrics in Service Catalog and dashboards
+
+**Symptom**: Metrics are not appearing in the Service Catalog and dashboards despite being properly collected. This typically occurs due to incorrect or improperly mapped semantic conventions.
```

Comment:
This section mentions proper semantic conventions several times, but it feels odd that it doesn't give an indication about how to find them. Can you make the assumption that customers reading this should already have those conventions at hand, or should this section link out to them just in case?